certbot/constants.py

```diff
@@ -95,3 +95,6 @@
 
 RENEWER_CONFIG_FILENAME = "renewer.conf"
 """Renewer config file name (relative to `IConfig.config_dir`)."""
+
+FORCE_INTERACTIVE_FLAG = "--force-interactive"
```

Comment:
This pattern is okay, but I'd note that we haven't used it in other situations where there are string literals around the code that contain things like this (`git grep cli_flag`)